crates/uv-python/src/interpreter.rs

```diff
@@ -439,9 +441,13 @@ impl Interpreter {
     }
 
     /// Return the `sys.path` for this Python interpreter.
-    pub fn sys_path(&self) -> &Vec<PathBuf> {
+    pub fn sys_path(&self) -> &[PathBuf] {
         &self.sys_path
     }
+    /// Return the `site.getsitepackages` for this Python interpreter.
+    pub fn site_packages_(&self) -> &[PathBuf] {
+        &self.site_packages
+    }
```

Comment:
This gets renamed and documented more thoroughly in https://github.com/astral-sh/uv/pull/15121